release-content/0.14/release-notes/10690_Upgrade_to_Taffy_04.md

```diff
@@ -1 +1,5 @@
-TODO
+[`taffy`](https://crates.io/crates/taffy), the UI layout crate that we rely on, has had a major update.
+In addition to various bug fixes, this brings two new pieces of functionality to users of `bevy_ui`:
+
+1. `Display::Block`: use the [CSS block layout model](https://www.w3schools.com/css/css_display_visibility.asp) to layout UI nodes. Elements will start on a new line, and take up the whole width. Great for in-line images!
+2. `Overflow::Hidden`: if elements overflow their container, they will be completely hidden, rather than shown or clipped.
```

Comment:
There isn't really any difference between "hidden" and "clipped" in this sentence. In Taffy difference between `overflow: hidden` and `overflow: clip` is primarily that `overflow: hidden` turns off the automatic minimum size of a flexbox or grid child. Again, the main benefit here is that `overflow: hidden` will match `overflow: hidden` on the web (`overflow: clip` is arguably often what you want but isn't widely used on the web because it's much newer than "hidden").